### PR TITLE
#3789 - Fix program description on Institution PIR Page

### PR DESCRIPTION
**Pull request by dheepak-aot**
## Issue description and fix

Issue: When a program is selected in a student application but study period is not selected and submitted for PIR review, the Institution PIR page was displaying the `program name` in the `program description` section.

- [x] Fix: The formIO was neither receiving nor displaying a program description that was coming from the original program. Hence the API was enhanced and FormIO was updated to receive the program description variable. 

image

image

### CODE REVIEW

File: sources/packages/backend/apps/api/src/services/application/application.service.ts [hunk 1]
```diff
@@ -522,6 +522,7 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "user.lastName",
         "pirProgram.id",
         "pirProgram.name",
+        "pirProgram.description",
```
Comment (guru-aot) [inline sources/packages/backend/apps/api/src/services/application/application.service.ts:525]: 👍

File: sources/packages/forms/src/form-definitions/programinformationrequest.json [hunk 2]
```diff
@@ -627,7 +629,7 @@
               "value": ""
             }
           ],
-          "content": "<!--studentCustomProgramDescription value is from the application dynamic data(text field) \nand studentSelectedProgram is from the program drop down-->\n{{ data.studentCustomProgramDescription ?? data.studentSelectedProgram }}",
+          "content": "<!--studentCustomProgramDescription value is from the application dynamic data(text field) \nand studentSelectedProgram is from the program drop down-->\n{{ data.studentCustomProgramDescription ?? data.studentSelectedProgramDescription }}",
```
Comment (guru-aot) [inline sources/packages/forms/src/form-definitions/programinformationrequest.json:632]: 👍

**Review by guru-aot — APPROVED** [review]
LGTM, nice and quick fix @dheepak-aot

**Review by andrewsignori-aot — APPROVED** [review]
Great fix, looks good 👍

**Pull request merged**